src/components/DatePickerInline/DatePickerInlineBase.tsx

```diff
@@ -97,48 +131,62 @@ function DatePickerInlineBase(props: DatePickerInlineBaseProps) {
         };
     }, [componentStyles, date, dates, startDate]);
 
+    const renderMonthComponent = useCallback(
+        (index: number) => {
+            return (
+                <Month
+                    locale={locale}
+                    mode={mode}
+                    validRange={validRange}
+                    index={index}
+                    date={date}
+                    dates={dates}
+                    startDate={startDate}
+                    endDate={endDate}
+                    onPressDate={onPressDate}
+                    scrollMode={scrollMode}
+                    disableWeekDays={disableWeekDays}
+                    isDocked={isDocked}
+                />
+            );
+        },
+        [
+            locale,
+            mode,
+            validRange,
+            date,
+            dates,
+            startDate,
+            endDate,
+            onPressDate,
+            scrollMode,
+            disableWeekDays,
+            isDocked,
+        ],
+    );
+
+    const renderCalenderHeader = useCallback(() => {
+        return isDocked ? (
```

Comment:
prevent conditional statements when you can.

Reply:
removed @shashank1010